deno.json

```diff
@@ -13,7 +13,8 @@
     "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.2.3",
     "std/": "https://deno.land/std@0.159.0/",
     "twind": "https://esm.sh/twind@0.16.19",
-    "twind/": "https://esm.sh/twind@0.16.19/"
+    "twind/": "https://esm.sh/twind@0.16.19/",
+    "chart.js": "https://esm.sh/stable/chart.js@4.0.1/auto"
```

Comment:
thanks for the heads-up, will definitely take a look!
however, not using `/auto` would require us to register all known types, scales, datapoint, etc...

the [react charts](https://react-chartjs-2.js.org/#quickstart) library does this, for instance.

```tsx
import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
import { Doughnut } from "react-chartjs-2";

ChartJS.register(ArcElement, Tooltip, Legend);

<Doughnut data={...} />
```

Reply:
You are right. The trade-off is that the user must select and register the required components themselves. I prefer this method as it reduces the bundle size. (like https://gist.github.com/ayame113/c6e2e0f69dc9b05174330f2a229d0c5b#file-chartjschart-tsx-L22)

Reply:
thanks for providing the gist!  🙏

potentially it could also make sense to provide both variants.
this way, we'd have a dead-simple, yet more bloat-y version, and a more flexible but smaller version, which requires some more configuration, wdyt?